.devcontainer/devcontainer.json

```diff
@@ -9,33 +9,30 @@
       "VARIANT": "18"
     }
   },
-
-  // Set *default* container specific settings.json values on container create.
-  "settings": {},
-
-  // Add the IDs of extensions you want installed when the container is created.
-  "extensions": [
-    "dbaeumer.vscode-eslint",
-    "cschleiden.vscode-github-actions",
```

Comment:
Have extensions been removed intentionally?

Reply:
yes, we don't really use them